src/BlockReportSearch/Block/BlockUsage.php

```diff
@@ -47,7 +47,12 @@ public function __construct(
      */
     public function get_blocks(Parameters $params): array
     {
-        $this->posts_ids = $this->search->get_posts($params);
+        $this->posts_ids = wp_cache_get( 'block_report_post_ids', 'planet4_master_theme' );
+
+        if ( $this->posts_ids === false ) {
+            $this->posts_ids = $this->search->get_posts($params);
+            wp_cache_set( 'block_report_post_ids', $this->posts_ids, 'planet4_master_theme', 120 );
```

Comment:
This is a nice use case for cache, and we can probably do something similar in other places 👍🏻 

Isn't `120` too short? That's 2 minutes, right? I think we can afford to cache the blocks report data for almost a day before invalidating.